addons/mail/models/discuss/discuss_channel.py

```diff
@@ -296,6 +296,13 @@ def _subscribe_users_automatically(self):
             ]
             # sudo: discuss.channel.member - adding member of other users based on channel auto-subscribe
             self.env['discuss.channel.member'].sudo().create(to_create)
+        notifications = []
+        for channel in self:
+            for group in channel.group_ids:
+                notifications.append((group, 'mail.record/insert', {
+                    'Thread': channel._channel_basic_info()
+                }))
```

Comment:
Shouldn't this notif force is_pinned true for the channel to show correctly?

Example: in messaging menu, it won't be there until reload.

Reply:
Yes, you are right, I forgot about it

Reply:
forgot to push the update?